lib/preview_web/live/preview_live.ex

```diff
@@ -58,7 +58,8 @@ defmodule PreviewWeb.PreviewLive do
     if to_charlist(str) == io, do: "selected=selected"
   end
 
-  def print_file_contents(file_contents, ext) when ext in [".ex", ".exs"] do
+  def print_file_contents(file_contents, ext)
+      when ext in [".ex", ".exs", ".erl", ".hrl", ".escript"] do
```

Comment:
Let's special case `rebar.config` and `rebar.config.script` also.

Reply:
And `.app.src`.